TileStache/__init__.py

```diff
@@ -86,34 +89,37 @@ def parseConfigfile(configpath):
               ...
             }
           }
-        
+
         The full path to the file is significant, used to
         resolve any relative paths found in the configuration.
-        
+
         See the Caches module for more information on the "caches" section,
         and the Core and Providers modules for more information on the
         "layers" section.
     """
-    config_dict = json_load(urlopen(configpath))
-    
-    scheme, host, path, p, q, f = urlparse(configpath)
-    
-    if scheme == '':
-        scheme = 'file'
-        path = realpath(path)
-    
-    dirpath = '%s://%s%s' % (scheme, host, dirname(path).rstrip('/') + '/')
+    if type(configHandle) is dict:
```

Comment:
Yeah, but maybe it is best in a separate pull request. Use isinstance here for now.